src/elk-monitoring/pagopa/paywallet/dashboards/walletDashboard.ndjson

```diff
@@ -1,2 +1,110 @@
-{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":1,"missingRefCount":0,"missingReferences":[]}
\ No newline at end of file
+{
+  "attributes": {
```

Comment:
inline json in this file since this is a [ndjson](https://github.com/ndjson/ndjson-spec) file where each json is separated by a new line (take the kibana export api call raw output)